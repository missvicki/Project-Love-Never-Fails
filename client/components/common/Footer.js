import React from "react";
import { Menu, List } from "semantic-ui-react";

import AllMenu from "./AllMenu";
import "../../assets/styles/footer.css";

const Footer = () => (
  <div className="footer">
    <Menu fixed="bottom" borderless className="footer" stackable>
      <Menu.Menu position="left">
        <List className="listItem">
          <List.Item className="listItem">
            P.O BOX 0000, MBARARA-UGANDA
          </List.Item>
          <List.Item className="listItem">
            ADDRESS: PROJECT LOVE NEVER FAILS, MBARARA - UGANDA
          </List.Item>
          <List.Item className="listItem">
            EMAIL: plnf@projectloveneverfails.com
          </List.Item>
        </List>
      </Menu.Menu>
      <Menu.Menu position="right" className="footerItem">
        <AllMenu items="footerItems" />
      </Menu.Menu>
    </Menu>
  </div>
);

export default Footer;
