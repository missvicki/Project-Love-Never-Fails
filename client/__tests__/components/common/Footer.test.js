import React from "react";
import { shallow } from "enzyme";
import Footer from "../../../components/common/Footer";

describe("<Footer />", () => {
  const wrapper = shallow(<Footer />);
  it("should render Footer component without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
