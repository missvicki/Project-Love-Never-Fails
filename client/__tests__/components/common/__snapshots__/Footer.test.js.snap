// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render Footer component without crashing 1`] = `
<div
  className="footer"
>
  <Menu
    borderless={true}
    className="footer"
    fixed="bottom"
    stackable={true}
  >
    <MenuMenu
      position="left"
    >
      <List
        className="listItem"
      >
        <ListItem
          className="listItem"
        >
          P.O BOX 0000, MBARARA-UGANDA
        </ListItem>
        <ListItem
          className="listItem"
        >
          ADDRESS: PROJECT LOVE NEVER FAILS, MBARARA - UGANDA
        </ListItem>
        <ListItem
          className="listItem"
        >
          EMAIL: plnf@projectloveneverfails.com
        </ListItem>
      </List>
    </MenuMenu>
    <MenuMenu
      className="footerItem"
      position="right"
    >
      <AllMenu
        items="footerItems"
      />
    </MenuMenu>
  </Menu>
</div>
`;
